python/dolfinx/fem/forms.py

```diff
@@ -47,6 +48,7 @@ def __init__(
         ],
         ufcx_form=None,
         code: typing.Optional[str] = None,
+        module: typing.Optional[types.ModuleType] = None,
```

Comment:
Is types.ModuleType correct? Not what I expected.

Reply:
The module object is created in ffcx using importlib.util.module_from_spec. According to
https://docs.python.org/3/library/types.html#types.ModuleType, types.ModuleType appears to be the correct type for Python modules.